seqr/management/tests/load_rna_seq_outlier_tests.py

```diff
@@ -25,31 +25,31 @@ def test_command(self, mock_gzip_open, mock_open, mock_logger):
             'NA19675_D2\tENSG00000240361\tdetail2\t0.01\t0.13\t-3.1\n',
             'NA19675_D2\tENSG00000233750\tdetail1\t0.064\t0.0000057\t7.8\n',
             'NA19675_D3\tENSG00000233750\tdetail1\t0.064\t0.0000057\t7.8\n',
+            'NA19675_D4\tENSG00000233750\tdetail1\t0.064\t0.0000057\t7.8\n',
         ]
-        mock_open.return_value.__enter__.return_value.__iter__.return_value = ['NA19675_D2\tNA19675_1']
+        mock_open.return_value.__enter__.return_value.__iter__.return_value = ['NA19675_D4\tNA19678']
 
         with self.assertRaises(ValueError) as e:
-            call_command('load_rna_seq', RNA_FILE_ID)
-        self.assertEqual(str(e.exception), 'Unable to find matches for the following samples: NA19675_D2, NA19675_D3')
+            call_command('load_rna_seq_outlier', RNA_FILE_ID)
+        self.assertEqual(str(e.exception), 'Unable to find matches for the following samples: NA19675_D3, NA19675_D4')
```

Comment:
I'm not sure what this means. It is tested because we mock the data in `gzip.open` and then test that it parses that data appropriately, if it wasn't being called correctly the mock would not return anything. This is consistant with the existing tests for the other load rna manage command

Reply:
I mean to test if the parameter of gzip.open is expected. Does it make sense?

Reply:
yes, and as I said this is consistant with existing tests and that is overkill